dataland-frontend/tests/sharedUtils/components/UploadDocuments.ts

```diff
@@ -53,4 +53,7 @@ export const uploadReports = {
     cy.get(`[data-test="${reportName}ToUploadContainer"]`).should("not.exist");
     cy.get(`[data-test="${reportName}AlreadyUploadedContainer"]`).should("not.exist");
   },
+  assertUploadButtonDisappeared(): void {
```

Comment:
Is this util also used for the uploading of eu taxonomy documents? If so i would not expect the "upload document" button to disappear.

Reply:
No it is only used for the upload buttons in LKSG currently. It tests a functionality that we introduced for situations where it should not be possible to upload more than one document with a given upload button (the case for LKSG certificates but maybe also other frameworks in the future)